storagemanager/users/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from .forms import UserRegisterForm, UserUpdateForm, ProfileUpdateForm
+from .models import Profile
+
+
+def register(request):
+    if request.method == 'POST':
+        form = UserRegisterForm(request.POST)
+        if form.is_valid():
```

Comment:
I'm thinking about wrapping this whole `if` part into `@transaction.atomic` to be sure everything will be saved to db or nothing at all. What do you think?